tests/ZfrOAuth2Test/Server/Grant/AuthorizationGrantTest.php

```diff
@@ -244,7 +255,7 @@ public function testCanCreateTokenResponse($hasRefreshGrant)
 
         $response = $this->grant->createTokenResponse($request, new Client(), $owner);
 
-        $body = json_decode($response->getContent(), true);
+        $body = json_decode($response->getBody(), true);
```

Comment:
No - you need to pass either a `StreamInterface` instance, or a string reference to a *resource* that can be used to seed a `Stream` instance. So, do this instead:

```php
$stream = new Stream('php://temp');
$stream->write(json_encode(['foo' => 'bar']));
$response = new Response($stream);
```

If you'll be creating streams that represent strings regularly, create a "string" stream variant that accepts the string in the constructor. (We will likely add something like that to diactoros, or to a "streams" package.)